servers/services/src/main/java/org/projectnessie/services/impl/TreeApiImpl.java

```diff
@@ -425,13 +425,19 @@ public MergeResponse transplantCommitsIntoBranch(
         transplants = s.collect(Collectors.toList());
       }
 
+      if (Boolean.TRUE.equals(keepIndividualCommits) && transplants.size() > 1) {
+        // Message overrides are not meaningful when transplanting more than one commit.
```

Comment:
Note: I don't mind to change the the default message(s) in a Nessie release. The current default message isn't great, and it's not a documented behavior.

Reply:
Since the commit messages (at least those from Iceberg) are usually somewhat meaningless, i'm not sure it's worth to keep even those.

Reply:
good point. #5694